paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -259,12 +262,57 @@ internal class PaymentSheetViewModelTest {
         }
     }
 
+    @Test
+    fun `removePaymentMethod uses retrieved 'CustomerState' info, not 'CustomerConfiguration' info`() = runTest {
```

Comment:
These new test names are very heavily based on the implementation. Could we name them based on their behavior instead?